.pipelines/v2/ci-using-the-latest-winappsdk.yml

```diff
@@ -0,0 +1,50 @@
+trigger: none
+pr: none
+schedules:
+  - cron: "0 0 * * *" # every day at midnight
+    displayName: "Daily midnight Build"
+    branches:
+      include:
+        - main
+    always: false # only run if there's code changes!
+
+name: $(BuildDefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
```

Comment:
Hi @crutkas @DHowett Do we need 1es template here?

Reply:
Probably not. :)